.travis.yml

```diff
@@ -0,0 +1,37 @@
+language: python
+python:
+  - "2.7"
+  - "3.4"
+  - "3.5"
+  - "3.6"      # current default Python on Travis CI
+  - "3.7"
+  - "3.7-dev"  # 3.7 development branch
+  - "3.8-dev"  # 3.8 development branch
+  - "nightly"  # nightly build
+  # PyPy versions
+  - "pypy"   # currently Python 2.7.13, PyPy 7.1.1
+  - "pypy3"  # currently Python 3.6.1,  PyPy 7.1.1-beta0
+# commands to run before dependencies installation
+before_install:
+  - mkdir -p $HOME/bin
+  - curl -fsSL https://testspace-client.s3.amazonaws.com/testspace-linux.tgz | tar -zxvf- -C $HOME/bin
+  - testspace config url kdious.testspace.com
+# command to install dependencies
+install:
+  - pip install -r requirements.txt
+  - pip install .
+# command to run tests
+script:
+  - pytest --cov-report xml:coverage.xml --cov=libinsult/ --junitxml=results.xml
+after_script:
+ - export FOLDER=Python:$TRAVIS_PYTHON_VERSION
+ - |
+   if [ "$TRAVIS_PYTHON_VERSION" = "3.6" ]; then
+     testspace [$FOLDER]results.xml coverage.xml
+   else
+     testspace [$FOLDER]results.xml
+   fi
+notifications:
+  email:
+    - kdious@yahoo.com
+    - sunnywithclouds@gmail.com
```

Comment:
Add a new line at the end of the file:
https://stackoverflow.com/questions/5813311/no-newline-at-end-of-file

Reply:
Thanks, I added a new line and CI passed. Although when I view the original file, it shows my manually added new line not there. Searching the internet has references to only yaml files with a +. My next assumption is adding two manual new lines incase the first is somehow omitted.
<img width="519" alt="Screen Shot 2019-10-20 at 12 07 13 PM" src="https://user-images.githubusercontent.com/8934297/67164829-6ed75c00-f333-11e9-80f7-cf5cc6718743.png">